src/modules/design/property-panel/indextabs/general/sections/timer-event/timer-event.ts

```diff
@@ -160,4 +202,51 @@ export class TimerEventSection implements ISection {
     }
   }
 
+  private _createExtensionElement(): void {
+    const extensionValues: Array<IModdleElement> = [];
+
+    const extensionElements: IModdleElement = this._moddle.create('bpmn:ExtensionElements', {values: extensionValues});
+    this._businessObjInPanel.extensionElements = extensionElements;
+  }
+
+  private _createPropertiesElement(): void {
+    const extensionElement: IExtensionElement = this._businessObjInPanel.extensionElements;
+
+    const properties: Array<IProperty> = [];
+    const propertiesElement: IPropertiesElement = this._moddle.create('camunda:Properties', {values: properties});
+
+    extensionElement.values.push(propertiesElement);
+  }
+
+  private _createProperty(propertyName: string): void {
+    const propertiesElement: IPropertiesElement = this._getPropertiesElement();
+
+    const propertyObject: Object = {
```

Comment:
Use `object` instead of `Object`

```suggestion
    const propertyObject: object = {
```